### ISSUE TITLE
Garage Entrances: New Editable Feature Layer - Jeremy S.

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-data-tech#3809]
From Jeremy:

> Is it possible to add a new layer for the Parking Inventory Map titled “Garage_Entrance” with the same attribute fields as the “Parking_Zones” layer and one feature for drawing a red line. I will be mapping out the garage entrances for the CBD and surrounding area. The plan was to draw lines, similar to the parking zone lines, where the entrances are for the garages downtown, but on its own layer, that we can turn on or off. My goal is to complete the mapping by Thursday of next week.

**Comment by jaime-mckeown** [comment]
After talking with Jeremy about his request, I asked him is there was a reason for using a line instead of a point since points are much easier to see when there are already a ton of lines. He liked the idea, will create as a point layer.

**Comment by jaime-mckeown** [comment]
- Create new layer in FGDB based on Parking Zones schema.
- Added Domains
- Published in AGOL
- Setup View layer for editing since the layer needs to be shared to everyone.

**Comment by jaime-mckeown** [comment]
Emailed Jeremy a link to the View Layer.

**Comment by jaime-mckeown** [comment]
- Jeremy and I discussed the Web App, I shared my screen with him and went over how to edit online since he has always edited feature layers in Mxds.
- He asked for 2 new fields: Garage Name, Company Management
- I also had forgot to add the street segment layer that he needs for reference.
- Had to recreate the View Layer since I added 2 new fields to the feature layer. The Web Map / App are configured for editing.
